docs/.vitepress/vitepress/components/vp-demo.vue

```diff
@@ -107,18 +109,10 @@ const copyCode = async () => {
     <!-- danger here DO NOT USE INLINE SCRIPT TAG -->
     <p class="example-description" v-html="decodedDescription" />
     <div class="example">
-      <Codepen
-        ref="codepenRef"
-        :css="props.css"
-        :css-pre-processor="props.cssPreProcessor"
-        :html="props.html"
-        :js="props.js"
-        :js-pre-processor="props.jsPreProcessor"
-      />
       <div class="op-btns">
-        <ElTooltip :content="locale['edit-in-codepen']" :visible-arrow="false">
+        <ElTooltip :content="locale['edit-in-editor']" :visible-arrow="false">
```

Comment:
We should also update locale

Reply:
I updated it locally, but I didn't submit it because of gitignore .. i am so sorry